### PR TITLE
Archive Python prototype scheduler

### PR DESCRIPTION
**Pull request by jherbel**
Also remove it from the CI.

### CODE REVIEW

File: archive/v2/tests/rebot.xml
Comment (SoloJacobs) [inline archive/v2/tests/rebot.xml:1]: This seems to be wrong. The file here is actually used by the rust tests. This commit does not move, it only copies.

Comment (jherbel) [reply archive/v2/tests/rebot.xml:1]: I copied it on purpose, st. the tests of the Python scheduler could in principle still run. In a follow-up commit, I will move the original file into the tests directory of the scheduler

**Review by SoloJacobs — COMMENTED** [review]
Why should also consider whether we simply delete this... But we can postpone this, after we have decideed what to do with v1

**Comment by jherbel** [comment]
> Why should also consider whether we simply delete this... But we can postpone this, after we have decideed what to do with v1

Agreed, but for now, I let's archive it. We might still need this as a reference for some time.

**Review by SoloJacobs — APPROVED** [review]
LGTM

**Pull request merged**